cmd/omctl/cmd/directory_update.go

```diff
@@ -0,0 +1,52 @@
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+
+	"github.com/checkr/openmock"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var directoryUpdateCmd = &cobra.Command{
+	Use:   "update",
+	Short: "update remote directory",
+	Long:  "update remote directory with new / updated components",
+	Args:  cobra.MaximumNArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		// load local templates
+		localOpenMock := openmock.OpenMock{}
+		localOpenMock.ParseEnv()
+		localOpenMock.TemplatesDir = localDirectory
+
+		localOpenMock.SetupLogrus()
+		localOpenMock.SetupRepo()
+
+		err := localOpenMock.Load()
+		if err != nil {
+			logrus.Fatalf("%s: %s", "failed to load yaml templates for mocks", err)
+		}
+
+		// post the loaded templates at the openmock instance
+		templatesYaml := localOpenMock.ToYAML()
+		response, httpErr := http.Post(openMockURL+"/api/v1/templates", "application/yaml", bytes.NewReader(templatesYaml))
```

Comment:
Ah i see, it does post the definitions to the remote openmock. I agree with earlier comments that we should normalize the terminology to something like 'definitions'. i think we should rename this command to reflect that it's updating the definitions|behaviors|templates, `omctl directory update` seems vague to me